core/src/main/resources/org/elasticsearch/plugins/plugin-install.help

```diff
@@ -20,14 +20,20 @@ DESCRIPTION
 
     The notation of 'username/repository' refers to a github repository.
 
+    The argument can be an valid <url> which points to a download or file location for the plugin to be loaded from.
+
 EXAMPLES
 
-    plugin install elasticsearch-analysis-kuromoji
+    plugin install analysis-kuromoji
 
     plugin install elasticsearch/shield/latest
 
     plugin install lmenezes/elasticsearch-kopf
 
+    plugin install http://download.elasticsearch.org/elasticsearch/elasticsearch-analysis-kuromoji/elasticsearch-analysis-kuromoji-2.7.0.zip
+
+    plugin install file:///path/to/plugin/elasticsearch-analysis-kuromoji-2.7.0.zip
```

Comment:
Sometimes non java devs are confused by that and I don't think `//` is required. May be this?

```
      plugin install file:/path/to/plugin/elasticsearch-analysis-kuromoji-2.7.0.zip
or
      plugin install file:./path/to/plugin/elasticsearch-analysis-kuromoji-2.7.0.zip
```

Or may be adding a windows user example like this (not tested) ?

```
      plugin install file:c:/path/to/plugin/elasticsearch-analysis-kuromoji-2.7.0.zip
```